recipes/watcher/all/test_package/test_package.cpp

```diff
@@ -20,7 +20,11 @@ int main(int argc, char** argv) {
   std::thread([&]() { WATCHER_NAMESPACE::watcher::watch(".", show_event_json); }).detach();
   auto const time_until_death = std::chrono::seconds(3);
   std::this_thread::sleep_for(time_until_death);
+#ifdef WATCHER_DIE_WITH_PATH
```

Comment:
INFO: It's needed to support both 0.5.0 and older version. The method `watcher::die`has changed its signature and needed that workaround.